network-config-analyzer/NetworkPolicy.py

```diff
@@ -64,6 +64,10 @@ def full_name(self, config_name=None):
             res = str(self.namespace) + '/' + res
         if config_name:
             res = config_name + '/' + res
+        # TODO: consider if should support identical full names in different layers (platforms)
+        # for multi-layer support, differentiate between two policies with the same name in different layers
```

Comment:
It seems we would have to handle the case where a network config from within a query in a scheme file refers to a single specific policy of some defined network config, currently it is specified by: <config>/<namespace>/<policy>. 
But if it is a policy name that exists in both k8s and calico, it would not be sufficient to identify which policy it refers to, would also have to add some 'layer' info, such as <config>/<namespace>/<policy>/<layer>.

Reply:
Added support for network policies with similar names on different layers.